LIBRARY.CSS

```diff
@@ -0,0 +1,100 @@
+
+/** THIS LIBRARY BUILD USUING CSS BLOCK CREATING TRICKS, CSS POSITIONING CSS FLEX. **/
+
+.CONTAINER{
+    width: 900px;
+    height: 900pX;
+    position: relative;
+    }
+
+.top{
+  width: 250px;
+  height: 30px;
+  background:mediumpurple;
+  border-radius: 20%;
+  position: absolute;
+  top:-25PX;  
+  left: 30px;
+}
+
+.LEFTBLOCK{
```

Comment:
As well as lowercasing this, you should use camelCase (not my preference, but you have followed this for the flex-type selectors below, so this will keep everything consistent), so that the selector ultimately becomes `.leftBlock`; this improves readability, and is a common convention.